### PR TITLE
upgraded the packages to latest versions available

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
### Description
The website had packages that were outdated so an issue to upgrade the package was raised and assigned to me

### Files changed
- public/mockServiceWorker.js (It is a library code that was changed automatically as soon as the library was upgraded)
- package.json
- yarn.lock

### Commands used
- `yarn upgrade --latest`
- using `yarn upgrade` did not upgrade to the latest packages to had to use the latest flag

### Is the development tested
- [x] Yes
- [ ] No

#### Other issues
- Found an error on DS `(/challenge)` page 
image

The errors are handled gracefully in prod but are not handled in dev. (Found out during the discussion call with Randhir and Yash on 20 August around 6 PM IST)

This could be raised as a new issue.

### CODE REVIEW

File: public/mockServiceWorker.js [hunk 1]
```diff
@@ -2,22 +2,21 @@
 /* tslint:disable */
```
Comment (RitikJaiswal75) [inline public/mockServiceWorker.js:2]: This is a library code that is changed automatically on upgrading the library.

More on this here: https://github.com/mswjs/msw

**Comment by heyrandhir** [comment]
LGTM.

**Comment by yesyash** [comment]
LGTM

File: public/mockServiceWorker.js (hunk 1 above)
Comment (ankushdharkar) [reply public/mockServiceWorker.js:2]: Thanks for leaving this note. It helps :)

**Review by ankushdharkar — APPROVED** [review]

**Pull request merged**